src/odemis/driver/technolution.py

```diff
@@ -836,6 +836,7 @@ def _setResolution(self, resolution):
         resolution = eff_cell_size * self.parent._mppc.shape[0:2]
 
         return tuple(resolution.astype(int))
+        return tuple(int(i) for i in resolution)
```

Comment:
I think the second return is never reached the way it is written down now. Probably you forgot to delete the first one?

Reply:
This one is here by accident. I will put this in a separate pr.